deepgram/_types.py

```diff
@@ -27,6 +27,8 @@ class UpdateResponse(TypedDict):
 class Options(TypedDict, total=False):
     api_key: str
     api_url: str  # this URL should /not/ include a trailing slash
+    suppress_warnings: bool
+    raise_warnings_as_errors: bool
```

Comment:
`TypedDict`s do not have default values, and none of the other classes in this file have default values. Technically these types should have `Optional` around them, but very few of the other optional fields in this file use `Optional` so I left it off here as well. This is something we can clean up in the future.

We could also convert the TypedDicts to dataclasses (I prefer dataclasses to TypedDicts). Alternatively, we could use TypedDict's `Required` and `NotRequired` types defined in this pep: https://peps.python.org/pep-0655/

Reply:
I don't have a strong preference - do you think TypedDicts are marginally more performant, but dataclasses are more flexible? We do have an awful lot of TypedDicts already, so it makes sense to defer to that standard for now.

My interest in an explicit default value is mainly to make clear what the behavior will be when the user doesn't set anything. But I guess in this case, it's logical for warnings to be warnings by default... rather than being suppressed or errors.